pom.xml

```diff
@@ -58,10 +57,15 @@
       <version>3.1.0</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.4</version>
+    </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>pubsub-light</artifactId>
-      <version>1.6</version>
+      <version>1.13-SNAPSHOT</version>
```

Comment:
Needs to be a non-snapshot before you merge this, right?

Reply:
done